R-Tutorial/scripts/RStudio.md

```diff
@@ -0,0 +1,11 @@
+# The website for downloading the is:
+
+[RStudio](https://www.rstudio.com/products/rstudio/download/)
+
+# Once downloaded, the following steps are followed :
+
+![RStudio_img1](../images/RStudio_img1.jpg)
+
+![RStudio_img2](../images/RStudio_img2.jpg)
```

Comment:
Again, for the above two images, they only contain simple text which can be written down in the markdown file instead of using a picture to showcase the same. Please write the text and remove the images (Don't forget to do this for all such files! I just checked [this](https://github.com/TheAlgorithms/R/blob/0b654a492818586c8eaf1929d369e8f481b7c1fc/R-Tutorial/scripts/RStudio.md) now)